Flutter/calendar/appointments.md

```diff
@@ -824,7 +824,7 @@ class Meeting {
 Add an exception appointment that is changed or modified occurrence of the recurrence pattern appointment to the `dateSource` of the `SfCalendar`. To add a changed occurrence, ensure to set the [RecurrenceId](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/Appointment/recurrenceId.html) of that occurrence, and add the date of that occurrence to the [RecurrenceExceptionDates](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/Appointment/recurrenceExceptionDates.html) of recurrence pattern appointment. The `RecurrenceId` of the changed occurrence should hold the exact recurrence pattern appointment [Id](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/Appointment/id.html). We can get the type of appointment from the [appointmentType](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/Appointment/appointmentType.html) property. 
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="16 25 28 29 30 31 32 33 34 36" %}
```

Comment:
https://help.syncfusion.com/flutter/calendar/appointments#delete-occurrence-from-recurrence-pattern-appointment-or-adding-exception-dates-to-recurrence-pattern-appointment

Not highlighted correctly

Reply:
Highlighted properly

Reply:
Not updated in pull request

Reply:
695 line highlighted properly.